cpp/custrings/tests/utils.h

```diff
@@ -1,8 +1,25 @@
 #include <cstdio>
 #include <vector>
 
+#include <rmm/rmm.h>
+
 #include "nvstrings/NVStrings.h"
 
+#define ASSERT_RMM_SUCCEEDED(expr)  ASSERT_EQ(RMM_SUCCESS, expr)
```

Comment:
@davidwendt review here

Reply:
looks good to me

Reply:
We should probably remove the `main` and use this fixture then